web/src/beta/components/Resizable/hooks.ts

```diff
@@ -104,7 +142,14 @@ export default (
     setPosition({ x: 0, y: 0 });
     setStartingSize(size);
     setDifference(0);
-  }, [isResizing, size]);
+    if (!localStorageKey) return;
```

Comment:
You actually don't need any of these returns I don't think since you are returning inside the `setLocalStorageData`